OpenBCI_32bit_Library.h

```diff
@@ -289,6 +289,10 @@ class OpenBCI_32bit_Library {
   void    LIS3DH_writeAxisDataForAxisSerial(uint8_t);
   void    LIS3DH_updateAxisData(void);
   void    LIS3DH_zeroAxisData(void);
+  char markerValue;
```

Comment:
this should go down to line 327 with the other `char`

Reply:
I see now - I put it with the function declarations!